🐛 [Bug] Windows dynamo core tests failed

##  Bug Description
https://github.com/pytorch/TensorRT/actions/runs/15912094165/job/44885062410

E0626 23:15:06.340000 3772 site-packages\torch\_guards.py:362] [2/0]   File "C:\actions-runner\_work\_temp\conda_environment_15912094165\Lib\site-packages\torch\_inductor\cpp_builder.py", line 147, in get_cpp_compiler

E0626 23:15:06.340000 3772 site-packages\torch\_guards.py:362] [2/0]     check_compiler_exist_windows(compiler)

E0626 23:15:06.340000 3772 site-packages\torch\_guards.py:362] [2/0]   File "C:\actions-runner\_work\_temp\conda_environment_15912094165\Lib\site-packages\torch\_inductor\cpp_builder.py", line 138, in check_compiler_exist_windows

E0626 23:15:06.340000 3772 site-packages\torch\_guards.py:362] [2/0]     raise RuntimeError(f"Compiler: {compiler} is not found.") from exc

E0626 23:15:06.340000 3772 site-packages\torch\_guards.py:362] [2/0] RuntimeError: Compiler: cl is not found.


## To Reproduce

Steps to reproduce the behavior:

1.
2.
3.

<!-- If you have a code sample, error messages, stack traces, please provide it here as well -->

## Expected behavior

<!-- A clear and concise description of what you expected to happen. -->

## Environment

> Build information about Torch-TensorRT can be found by turning on debug messages

 - Torch-TensorRT Version (e.g. 1.0.0):
 - PyTorch Version (e.g. 1.0):
 - CPU Architecture:
 - OS (e.g., Linux):
 - How you installed PyTorch (`conda`, `pip`, `libtorch`, source):
 - Build command you used (if compiling from source):
 - Are you using local sources or building from archives:
 - Python version:
 - CUDA version:
 - GPU models and configuration:
 - Any other relevant information:

## Additional context

<!-- Add any other context about the problem here. -->

python -m pytest -k different -rP C:\Users\lanl\git\TensorRT\tests\py\dynamo\runtime\test_mutable_torchtrt_module.py

I have installed the cl.exe in windows:
```
(venv_py310) C:\Users\lanl>where cl.exe
C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.44.35207\bin\Hostx64\x64\cl.exe

```
got the different error:
```
git\TensorRT\tests\py\dynamo\runtime\test_mutable_torchtrt_module.py:652:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
git\venv_py310\lib\site-packages\torch_tensorrt\dynamo\runtime\_MutableTorchTensorRTModule.py:538: in __call__
    return self.forward(*args, **kwargs)
git\venv_py310\lib\site-packages\torch_tensorrt\dynamo\runtime\_MutableTorchTensorRTModule.py:491: in forward
    self.compile()
git\venv_py310\lib\site-packages\torch_tensorrt\dynamo\runtime\_MutableTorchTensorRTModule.py:360: in compile
    self.exp_program = self.get_exported_program()
git\venv_py310\lib\site-packages\torch_tensorrt\dynamo\runtime\_MutableTorchTensorRTModule.py:349: in get_exported_program
    return export_fn()
git\venv_py310\lib\site-packages\torch_tensorrt\dynamo\runtime\_MutableTorchTensorRTModule.py:325: in export_fn
cmd_line = 'cl /I "C:/Users/lanl/AppData/Local/Programs/Python/Python310/Include" /I "C:/Users/lanl/git/venv_py310/lib/site-packa...:/Users/lanl/git/venv_py310/lib/site-packages/torch/lib"  "torch.lib" "torch_cpu.lib" "torch_python.lib" "sleef.lib"  '
cwd = 'C:\\Users\\lanl\\AppData\\Local\\Temp\\torchinductor_lanl\\vx\\cvxvdk5bvkz4k7bxtihzucsv7vdcskamswje43lmzvmbfbrmypse.main_CxxBuild'

    def _run_compile_cmd(cmd_line: str, cwd: str) -> None:
        cmd = shlex.split(cmd_line)
        try:
            subprocess.run(
                cmd, cwd=cwd, check=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT
            )
        except subprocess.CalledProcessError as e:
            output = e.stdout.decode("utf-8")
            openmp_problem = "'omp.h' file not found" in output or "libomp" in output
            if openmp_problem and sys.platform == "darwin":
                instruction = (
                    "\n\nOpenMP support not found. Please try one of the following solutions:\n"
                    "(1) Set the `CXX` environment variable to a compiler other than Apple clang++/g++ "
                    "that has builtin OpenMP support;\n"
                    "(2) install OpenMP via conda: `conda install llvm-openmp`;\n"
                    "(3) install libomp via brew: `brew install libomp`;\n"
                    "(4) manually setup OpenMP and set the `OMP_PREFIX` environment variable to point to a path"
                    " with `include/omp.h` under it."
                )
                output += instruction
>           raise exc.CppCompileError(cmd, output) from e
E           torch._dynamo.exc.InternalTorchDynamoError: CppCompileError: C++ compile error
E
E           Command:
E           cl /I C:/Users/lanl/AppData/Local/Programs/Python/Python310/Include /I C:/Users/lanl/git/venv_py310/lib/site-packages/torch/include /I C:/Users/lanl/git/venv_py310/lib/site-packages/torch/include/torch/csrc/api/include /D TORCH_INDUCTOR_CPP_WRAPPER /D STANDALONE_TORCH_HEADER /D C10_USING_CUSTOM_GENERATED_MACROS /DLL /MD /O2 /std:c++20 /wd4819 /wd4251 /wd4244 /wd4267 /wd4275 /wd4018 /wd4190 /wd4624 /wd4067 /wd4068 /EHsc /openmp /openmp:experimental C:/Users/lanl/AppData/Local/Temp/torchinductor_lanl/vx/cvxvdk5bvkz4k7bxtihzucsv7vdcskamswje43lmzvmbfbrmypse.main.cpp /FeC:/Users/lanl/AppData/Local/Temp/torchinductor_lanl/vx/cvxvdk5bvkz4k7bxtihzucsv7vdcskamswje43lmzvmbfbrmypse.main.pyd /LD /link /LIBPATH:C:/Users/lanl/AppData/Local/Programs/Python/Python310/libs /LIBPATH:C:/Users/lanl/git/venv_py310/lib/site-packages/torch/lib torch.lib torch_cpu.lib torch_python.lib sleef.lib
E
E           Output:
E           Microsoft (R) C/C++ Optimizing Compiler Version 19.44.35211 for x64
E           Copyright (C) Microsoft Corporation.  All rights reserved.
E
E           cl : Command line warning D9025 : overriding '/openmp' with '/openmp:experimental'
E           cvxvdk5bvkz4k7bxtihzucsv7vdcskamswje43lmzvmbfbrmypse.main.cpp
E           C:/Users/lanl/AppData/Local/Temp/torchinductor_lanl/vx/cvxvdk5bvkz4k7bxtihzucsv7vdcskamswje43lmzvmbfbrmypse.main.cpp(2): fatal error C1083: Cannot open include file: 'algorithm': No such file or directory

git\venv_py310\lib\site-packages\torch\_inductor\cpp_builder.py:396: InternalTorchDynamoError
```

--- a/.github/workflows/build-test-tensorrt-windows.yml
+++ b/.github/workflows/build-test-tensorrt-windows.yml
@@ -256,7 +256,7 @@ jobs:
         cd dynamo
         ../../../packaging/vc_env_helper.bat python -m pytest -ra -n 4 --junitxml=${RUNNER_TEST_RESULTS_DIR}/tests_py_dynamo_core_runtime_test_results.xml --ignore runtime/test_002_cudagraphs_py.py --ignore runtime/test_002_cudagraphs_cpp.py runtime/
         python -m pytest -ra -n 4 --junitxml=${RUNNER_TEST_RESULTS_DIR}/tests_py_dynamo_core_partitioning_test_results.xml partitioning/
-        python -m pytest -ra -n 4 --junitxml=${RUNNER_TEST_RESULTS_DIR}/tests_py_dynamo_core_lowering_test_results.xml lowering/
+        ../../../packaging/vc_env_helper.bat python -m pytest -ra -n 4 --junitxml=${RUNNER_TEST_RESULTS_DIR}/tests_py_dynamo_core_lowering_test_results.xml lowering/
         popd
 
   tests-py-dynamo-cudagraphs:

--- a/.github/workflows/build-test-tensorrt-windows.yml
+++ b/.github/workflows/build-test-tensorrt-windows.yml
@@ -254,7 +254,7 @@ jobs:
         pushd .
         cd tests/py
         cd dynamo
-        python -m pytest -ra -n 4 --junitxml=${RUNNER_TEST_RESULTS_DIR}/tests_py_dynamo_core_runtime_test_results.xml --ignore runtime/test_002_cudagraphs_py.py --ignore runtime/test_002_cudagraphs_cpp.py runtime/
+        ../../../packaging/vc_env_helper.bat python -m pytest -ra -n 4 --junitxml=${RUNNER_TEST_RESULTS_DIR}/tests_py_dynamo_core_runtime_test_results.xml --ignore runtime/test_002_cudagraphs_py.py --ignore runtime/test_002_cudagraphs_cpp.py runtime/
         python -m pytest -ra -n 4 --junitxml=${RUNNER_TEST_RESULTS_DIR}/tests_py_dynamo_core_partitioning_test_results.xml partitioning/
         python -m pytest -ra -n 4 --junitxml=${RUNNER_TEST_RESULTS_DIR}/tests_py_dynamo_core_lowering_test_results.xml lowering/
         popd

--- a/.github/workflows/build-test-windows.yml
+++ b/.github/workflows/build-test-windows.yml
@@ -260,7 +260,7 @@ jobs:
         cd dynamo
         ../../../packaging/vc_env_helper.bat python -m pytest -ra -n 4 --junitxml=${RUNNER_TEST_RESULTS_DIR}/tests_py_dynamo_core_runtime_test_results.xml --ignore runtime/test_002_cudagraphs_py.py --ignore runtime/test_002_cudagraphs_cpp.py runtime/
         python -m pytest -ra -n 4 --junitxml=${RUNNER_TEST_RESULTS_DIR}/tests_py_dynamo_core_partitioning_test_results.xml partitioning/
-        python -m pytest -ra -n 4 --junitxml=${RUNNER_TEST_RESULTS_DIR}/tests_py_dynamo_core_lowering_test_results.xml lowering/
+        ../../../packaging/vc_env_helper.bat python -m pytest -ra -n 4 --junitxml=${RUNNER_TEST_RESULTS_DIR}/tests_py_dynamo_core_lowering_test_results.xml lowering/
         popd
 
   tests-py-dynamo-cudagraphs:

--- a/.github/workflows/build-test-windows.yml
+++ b/.github/workflows/build-test-windows.yml
@@ -258,7 +258,7 @@ jobs:
         pushd .
         cd tests/py
         cd dynamo
-        python -m pytest -ra -n 4 --junitxml=${RUNNER_TEST_RESULTS_DIR}/tests_py_dynamo_core_runtime_test_results.xml --ignore runtime/test_002_cudagraphs_py.py --ignore runtime/test_002_cudagraphs_cpp.py runtime/
+        ../../../packaging/vc_env_helper.bat python -m pytest -ra -n 4 --junitxml=${RUNNER_TEST_RESULTS_DIR}/tests_py_dynamo_core_runtime_test_results.xml --ignore runtime/test_002_cudagraphs_py.py --ignore runtime/test_002_cudagraphs_cpp.py runtime/
         python -m pytest -ra -n 4 --junitxml=${RUNNER_TEST_RESULTS_DIR}/tests_py_dynamo_core_partitioning_test_results.xml partitioning/
         python -m pytest -ra -n 4 --junitxml=${RUNNER_TEST_RESULTS_DIR}/tests_py_dynamo_core_lowering_test_results.xml lowering/
         popd

